components/jaggery-core/org.jaggeryjs.jaggery.core/src/main/resources/META-INF/scripts/http.js

```diff
@@ -14,7 +14,7 @@ var get, post, put, del, head, options, trace, connect;
         if (type === "xml") {
 			return new XML(data.replace(/<\?xml.*?\?>/, "").replace(/<!--[\s\S]*?-->/g, ""));
 		} else if (type === "json") {
-			return parse(data);
+			return JSON.parse(data);
```

Comment:
This jaggery module followed tab indentation instead of 4 space indentation as it was flowed at the time module was written.